great-idea-website/css/index.css

```diff
@@ -54,13 +54,155 @@ table {
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
+	
 }
 
 h1, h2, h3, h4, h5 {
-    font-family: 'Bangers', cursive;
+	font-family: 'Bangers', cursive;
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body{
+	width: 100%;
+	padding: 10px;
+	border: 1px thin white;
+	font-family: 'Nunito', sans-serif;
+}
+h1 {
+	font-family: 'Noto Serif', serif;
+	letter-spacing: 3px;
+	font-weight: bolder;
+	font-size: 30px
+}
+header{
+	display: inline-block;
+	width: 100%;
+	margin: 10px 10px;
+	padding: 10px 20px;
+	vertical-align: middle;
+}
+nav{
+	border: 1px solid lavenderblush;
+	height: 50px;
+	display: inline-block;
+	background-color: lightgoldenrodyellow;
+	position: fixed;
+	z-index: +1;
+	width: 100%;
+	right: 5px;
+	top:0
+}
```

Comment:
I love that you was able to give a fixed position to the nav bar. It looks like the position of the elements in it is a bit off. 
You could try to work more on `header div {` to move the logo to the right.